src/core/global.hpp

```diff
@@ -1,88 +0,0 @@
-/*
```

Comment:
With this file gone, we can finally remove external linkage of time-related global variables: f78df9f8b7bacf66e315b40ffdc7efc3fafb67d4

Reply:
This will be addressed in another PR, many more globals can now be made static.